### PR TITLE
Feat/mp 2075 update templates

### PR DESCRIPTION
**Pull request by germainruget**
[MP-2075](https://lumapps.atlassian.net/browse/MP-2075?atlOrigin=eyJpIjoiNjA4OTEwNjNkNTA3NGVhZDgxYzAzMGUzZTE2MDY2MmUiLCJwIjoiaiJ9)

Update dependencies
Update widget template to use useProperties hooks

[MP-2075]: https://lumapps.atlassian.net/browse/MP-2075?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: template-widget-extension/src/widget/WidgetSettings.tsx [hunk 1]
```diff
@@ -60,19 +59,24 @@ const WithIntlSettings: WidgetSettings = ({ properties = {}, exportProp }) => {
     );
 };
 
-export const WidgetSettings: WidgetSettings = (props) => {
+export const WidgetSettings: SettingsProps = (props) => {
```
Comment (LaurentMox) [inline template-widget-extension/src/widget/WidgetSettings.tsx:62]: Props seems unused

File: template-widget-extension/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "template-react-ts",
-  "version": "1.2.5",
+  "version": "1.3.0",
   "private": true,
   "scripts": {
```
Comment (LaurentMox) [inline template-widget-extension/package.json:5]: Look like there is some missing dependencies when running these jobs

**Review by LaurentMox — CHANGES_REQUESTED** [review]

**Comment by LaurentMox** [comment]
+ need to remove *.trans.* files

File: template-widget-extension/src/widget/WidgetSettings.tsx (hunk 1 above)
Comment (germainruget) [reply template-widget-extension/src/widget/WidgetSettings.tsx]: Props removed 👍

File: template-widget-extension/package.json (hunk 2 above)
Comment (germainruget) [reply template-widget-extension/package.json:5]: The scripts seems to work correctly on my side

**Comment by gaetanDev7** [comment]
## **For the template widget** 

dependencies installation : OK 

widget launch : OK

launch test :  OK 

The extension has some warning on yarn start :
image

Some linter error when execute yarn lint. the linter should note see the coverage files 
image

## **For the template Share** 

dependencies installation : OK 

widget launch : OK

launch test :  OK 

some linter error 

image

## **For the template Backend ** 

dependencies installation : OK 

widget launch : OK

launch test :  OK 

some linter error 
image

## **For the template search  ** 

dependencies installation :  

widget launch : 

launch test :   

some linter error 
... in progress .....


## **For the template Empty  **

**Review by gaetanDev7 — DISMISSED** [review]
# Code Summary

LGTM

Need to replace => "@lumapps-extensions-playground/devenv": "1.26.0-beta.1" after publishing the release

Code ${\color{green}OK}$

# Test Summary

ALL : ${\color{green}OK}$

## For the template widget

extension generated propertly : ${\color{green}OK}$

dependencies installation : ${\color{green}OK}$

widget launch : ${\color{green}OK}$

extension works properly : ${\color{green}OK}$

launch test : ${\color{green}OK}$

linter : ${\color{green}OK}$

build : ${\color{green}OK}$

## For the template share

extension generated propertly : ${\color{green}OK}$

dependencies installation : ${\color{green}OK}$

widget launch : ${\color{green}OK}$

extension works properly : ${\color{green}OK}$

launch test : ${\color{green}OK}$

linter : ${\color{green}OK}$

build : ${\color{green}OK}$

## For the template backend

extension generated propertly : ${\color{green}OK}$

dependencies installation : ${\color{green}OK}$

widget launch : ${\color{green}OK}$

extension works properly : ${\color{green}OK}$

launch test : ${\color{green}OK}$

linter : ${\color{green}OK}$

build : ${\color{green}OK}$

## For the template search

extension generated propertly : ${\color{green}OK}$

dependencies installation : ${\color{green}OK}$

widget launch : ${\color{green}OK}$

extension works properly : ${\color{green}OK}$

image

launch test : ${\color{green}OK}$

linter : ${\color{green}OK}$

build : ${\color{green}OK}$

## For the template empty

extension generated propertly : ${\color{green}OK}$

dependencies installation : ${\color{green}OK}$

widget launch : ${\color{green}OK}$

extension works properly : ${\color{green}OK}$

launch test : ${\color{green}OK}$

linter : ${\color{green}OK}$

build : ${\color{green}OK}$

**Pull request merged**